binaries/assimp/include/assimp/material.inl

```diff
@@ -98,7 +98,7 @@ AI_FORCE_INLINE aiReturn aiMaterial::Get(const char* pKey,unsigned int type,
             return AI_FAILURE;
         }
 
-        iNum = static_cast<unsigned int>(min(static_cast<size_t>(iNum),prop->mDataLength / sizeof(Type)));
+        iNum = static_cast<unsigned int>(std::min<size_t>(static_cast<size_t>(iNum),prop->mDataLength / sizeof(Type)));
```

Comment:
Is this fixing a warning or an error? If this is just a warning, I'd rather silence it. Since this is an external file, we risk losing the change if we update the Assimp version.

Reply:
This fix is compile error on Ubuntu 22.04

Reply:
I also checked assimp github and they have fixed the same line in at least in the latest tagged release 👌